fpl/data/data_converter.py

```diff
@@ -0,0 +1,79 @@
+"""Convert data to CSV."""
+import json
+from pathlib import Path
+
+import pandas as pd
+from tqdm import tqdm
+
+from fpl.utils import paths
+
+
+class DataConverter:
+    """Data converter class."""
+
+    def __init__(self, entity: str, raw_data_path=None, interim_data_path=None):
+        """Set configuration parameters.
+
+        Args:
+            entity (str): Either "elements" or "teams".
+            raw_data_path (Path, optional): Path to dir holding JSON dumps of raw data.
+            interim_data_path (Path, optional): Path to dir for uploading converted data.
+        """
+        self.entity = entity
+        if raw_data_path is None:
```

Comment:
This could perhaps be shortened to:
```python
self.__raw_data_path = Path(raw_data_path) if raw_data_path else paths.get_raw_data_path()
```
Consider readability. Goes for the rest of the assignments.
https://docs.python.org/3/reference/expressions.html#conditional-expressions